### PR TITLE
unit tests for number formatter components

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
add or expand unit tests for MoneyRange, FormatNumber, FormatNumberRange
add THumanReadableFormat, split EHumanReadableFormat in normal and abbreviated form enums

### CODE REVIEW

File: app/components/eto/registration/pages/etoTermsUtils.tsx [hunk 1]
```diff
@@ -4,13 +4,13 @@ import { FormattedMessage } from "react-intl-phraseapp";
 import { EProductName } from "../../../../lib/api/eto/EtoProductsApi.interfaces";
 import { assertNever } from "../../../../utils/assertNever";
 import { invariant } from "../../../../utils/invariant";
-import { EHumanReadableFormat, ToHumanReadableForm } from "../../../shared/ToHumanReadableForm";
```
Comment (Moejoe90) [inline app/components/eto/registration/pages/etoTermsUtils.tsx]: `THumanReadableFormat` does not follow our enum naming rules

File: app/components/eto/registration/pages/etoTermsUtils.tsx [hunk 2]
```diff
@@ -4,13 +4,13 @@ import { FormattedMessage } from "react-intl-phraseapp";
...
+import { THumanReadableFormat, ToHumanReadableForm } from "../../../shared/ToHumanReadableForm";
```
Comment (Moejoe90) [inline app/components/eto/registration/pages/etoTermsUtils.tsx]: Is this an enum `THumanReadableFormat`? it should start with an

Comment (AlexeyTuboltsev) [reply app/components/eto/registration/pages/etoTermsUtils.tsx]: no, it's a type that is a union of two enums

**Comment by AlexeyTuboltsev** [comment]
please let Pawel merge #2884 first, otherwise he'll have a huge bunch of conflicts

**Pull request merged**